js/main.js

```diff
@@ -1,2 +1,82 @@
-$(function () {
+$(document).ready(function () {
+  let userScore = 0
+  let botScore = 0
+  const choices = ["rock", "paper", "scissors"]
+
+
+  $("#rock").click(() => {
+    compare("rock", generate())
+  })
+  $("#paper").click(() => {
+    const bot = generate();
+    compare("paper", bot)
+  })
+  $("#scissors").click(() => {
+    const user = "scissors"
+    const bot = generate()
+    compare(user, bot)
+  })
```

Comment:
Odellya, really great use of passing variables as arguments for functions you call on click. It is best practice to separate code that have different functions (which you have clearly done). Nice job!